resources/testharness.js

```diff
@@ -2112,10 +2155,11 @@ policies and contribution forms [3].
 
     Tests.prototype.result = function(test)
     {
-        if (this.phase > this.phases.HAVE_RESULTS) {
-            return;
+        // If the harness has already transitioned beyond the `HAVE_RESULTS`
+        // phase, subsequent tests should not cause it to revert.
+        if (this.phase <= this.phases.HAVE_RESULTS) {
```

Comment:
Micronit: `<=` can be `<` to avoid the appearance of doing nothing.

Reply:
This code was also factored away.